app/models/street/StreetEdgeTable.scala

```diff
@@ -196,42 +196,38 @@ object StreetEdgeTable {
       }
 
       // Get length of each street segment, sum the lengths, and convert from meters to miles.
-      edgesWithAuditCounts.filter(_._2 >= auditCount).map(_._1).sum.run.map(_ * 0.000621371F).getOrElse(0.0F)
+      return edgesWithAuditCounts.filter(_._2 >= auditCount).map(_._1).sum.run.map(_ * 0.000621371F).getOrElse(0.0F)
     }
   }
 
   /**
-    * Calculates the distance audited today by all users.
-    *
-    * @return The distance audited today by all users in miles.
-    */
-  def auditedStreetDistanceToday(): Float = db.withSession { implicit session =>
-    val getDistanceQuery = Q.queryNA[Float](
-      """SELECT SUM(ST_Length(ST_Transform(geom, 26918)))
-        |FROM street_edge
-        |INNER JOIN audit_task ON street_edge.street_edge_id = audit_task.street_edge_id
-        |WHERE (audit_task.task_end AT TIME ZONE 'US/Pacific')::date = (now() AT TIME ZONE 'US/Pacific')::date
-        |     AND street_edge.deleted = FALSE
-        |     AND audit_task.completed = TRUE""".stripMargin
-    )
-    (getDistanceQuery.first * 0.000621371).toFloat;
-  }
+   * Calculates the total distance audited by all users over a specified time period.
+   *
+   * @param timeInterval can be "today" or "week". If anything else, defaults to "all time".
+   * @return The total distance audited by all users in miles.
+   */
+  def auditedStreetDistanceOverTime(timeInterval: String = "all time"): Float = db.withSession { implicit session =>
+    val cacheKey = s"auditedStreetDistanceOverTime($timeInterval)"
+    Cache.getOrElse(cacheKey, 30.minutes.toSeconds.toInt) {
```

Comment:
I actually don't think we should bother caching this value. We only use this function on the admin page, so I'd rather have fully up-to-date stats than get a small speedup just for admins.

Reply:
Done!